src/components/LineChart/LineChart.js

```diff
@@ -114,100 +115,105 @@ function LineChart({
-          height="auto"
-          css="display: block"
-          {...props}
-        >
-          <mask id="chart-mask">{rectangle}</mask>
-          {rectangle}
-
-          <g mask="url(#chart-mask)">
-            {totalCount > 0 && (
-              <path
-                d={`
+      {/* {({ progress }) => { */}
+      {({ progress }) => {
+        const val = 1
+
+        return (
+          <svg
+            ref={onSvgRef}
+            viewBox={`0 0 ${width} ${height}`}
+            width={widthProps || 'auto'}
+            height="auto"
+            css="display: block"
+            {...props}
+          >
+            <mask id="chart-mask">{rectangle}</mask>
+            {rectangle}
+
+            <g mask="url(#chart-mask)">
+              {totalCount > 0 && (
+                <path
+                  d={`
                   ${[...new Array(totalCount - 1)].reduce(
                     (path, _, index) =>
                       `${path} M ${getX(index)},${chartHeight} l 0,-8`,
                     ''
                   )}
                 `}
-                stroke={borderColor}
-                strokeWidth="1"
-              />
-            )}
-            {lines.map((line, lineIndex) => (
-              <g key={`line-plot-${line.id || lineIndex}`}>
-                <path
-                  d={`
+                  stroke={borderColor}
+                  strokeWidth="1"
+                />
+              )}
+              {lines.map((line, lineIndex) => (
+                <g key={`line-plot-${line.id || lineIndex}`}>
+                  <path
+                    d={`
                     M
                     ${getX(0)},
-                    ${getY(line.values[0], progress, chartHeight)}
+                    ${getY(line.values[0], val, chartHeight)}
 
                     ${line.values
                       .slice(1)
                       .map(
                         (val, index) =>
                           `L
-                           ${getX((index + 1) * progress)},
-                           ${getY(val, progress, chartHeight)}
+                           ${getX((index + 1) * val)},
+                           ${getY(val, val, chartHeight)}
                           `
                       )
                       .join('')}
                   `}
-                  fill="transparent"
-                  stroke={line.color || color(lineIndex, { lines })}
-                  strokeWidth="2"
-                />
-                {line.values.slice(1, -1).map((val, index) => (
-                  <circle
-                    key={index}
-                    cx={getX(index + 1) * progress}
-                    cy={getY(val, progress, chartHeight)}
-                    r={dotRadius}
-                    fill="white"
+                    fill="transparent"
                     stroke={line.color || color(lineIndex, { lines })}
-                    strokeWidth="1"
+                    strokeWidth="2"
```

Comment:
we need increase the strokeWidth for some reason?

Reply:
For testing, I was playing with different values and I was not supposed to commit that file.